config/initializers/omniauth.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.config.middleware.use OmniAuth::Builder do
   provider :google_oauth2, Rails.application.credentials.dig(:google_oauth2, :client_id), Rails.application.credentials.dig(:google_oauth2, :client_secret)
-end
\ No newline at end of file
+  provider :intuit, Rails.application.credentials.dig(:intuit, :client_id), Rails.application.credentials.dig(:intuit, :client_secret), mode: :development, scope: "openid email profile"
```

Comment:
This should be in sandbox mode, not development mode.
```suggestion
  provider :intuit, Rails.application.credentials.dig(:intuit, :client_id), Rails.application.credentials.dig(:intuit, :client_secret), mode: :sandbox, scope: "openid email profile"
```